localstack/services/ses/provider.py

```diff
@@ -178,6 +181,29 @@ def create_configuration_set_event_destination(
 
         return result
 
+    @handler("DeleteConfigurationSet")
+    def delete_configuration_set(
+        self, context: RequestContext, configuration_set_name: ConfigurationSetName
+    ) -> DeleteConfigurationSetResponse:
+        # not implemented in moto
+        # TODO: contribute upstream?
+        backend = get_ses_backend(context)
+        backend.config_set.pop(configuration_set_name, None)
+        return {}
+
+    @handler("DeleteConfigurationSetEventDestination")
+    def delete_configuration_set_event_destination(
+        self,
+        context: RequestContext,
+        configuration_set_name: ConfigurationSetName,
+        event_destination_name: EventDestinationName,
+    ) -> DeleteConfigurationSetEventDestinationResponse:
+        # not implemented in moto
+        # TODO: contribute upstream?
+        backend = get_ses_backend(context)
+        backend.config_set_event_destination.pop(configuration_set_name, None)
```

Comment:
question: same question here, does it just returns if the `ConfigurationSet` or the `EventDestination` does not exist?